usaspending_api/download/helpers.py

```diff
@@ -198,17 +198,29 @@ def split_csv(file_path, delimiter=',', row_limit=10000, output_name_template='o
         `output_name_template`: A %s-style template for the numbered output files.
         `output_path`: Where to stick the output files.
         `keep_headers`: Whether or not to print the headers in each output file.
+        `file_name`: Alternative naming convention for output files (optional)
     Example usage:
         >> from toolbox import csv_splitter;
         >> csv_splitter.split('/home/ben/input.csv');
     """
     split_csvs = []
     reader = csv.reader(open(file_path, 'r'), delimiter=delimiter)
     current_piece = 1
-    current_out_path = os.path.join(
-        output_path,
-        output_name_template % current_piece
-    )
+    if file_name is not None:
+        zip_idx = file_name.find(".zip")
+
+        if(zip_idx > 0):
+            file_name = "%s.csv" % file_name[:zip_idx]
+
+        current_out_path = os.path.join(
```

Comment:
Do these still have numbers on the end of them? That's kinda a crucial feature and the premise of the bulk downloads. Basically for csvs with more than a million rows, we want to split those out to other csvs with the order preserved. All to say, it should end up something like, 

`download_results.zip`
 -> `download_results_1.csv`
 -> `download_results_2.csv`
etc.

Reply:
Oh, I understand now. The new commit now utilizes the numbering convention argument.